### PR TITLE
docs: improvement issue template

### PR DESCRIPTION
**Pull request by hifabienne**
### Definition of Ready

- [ ] I am happy with the code
- [ ] Short description of the feature/issue is added in the pr description
- [ ] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [ ] No debug or dead code
- [ ] My code has no repetitions
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/BUG_REPORT.yaml
Comment (eliobischof) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.yaml:1]: Is the file name BUG_REPORT.yaml some gh convention?

Comment (hifabienne) [reply .github/ISSUE_TEMPLATE/BUG_REPORT.yaml:1]: I don't know, it already existed so I just changed it.

File: .github/ISSUE_TEMPLATE/improvement.yaml
[the remark this answers is not in the document]
Comment (hifabienne) [reply .github/ISSUE_TEMPLATE/improvement.yaml:1]: Does that really make sense? At that point we do not really know if it is something that we want to include in zitadel

File: .github/ISSUE_TEMPLATE/BUG_REPORT.yaml
[the remark this answers is not in the document]
Comment (eliobischof) [reply .github/ISSUE_TEMPLATE/BUG_REPORT.yaml:1]: Don't really understand your point :joy: But I have absolutely no hard feelings if we omit it.

**Review by eliobischof — APPROVED** [review]

**Pull request merged**